protostar/testing/fuzzing/strategies/short_strings.py

```diff
@@ -0,0 +1,26 @@
+import string
+
+from hypothesis.strategies import SearchStrategy, text
+from starkware.cairo.lang.compiler.ast.cairo_types import CairoType, TypeFelt
+
+from protostar.testing.fuzzing.exceptions import SearchStrategyBuildError
+from protostar.testing.fuzzing.strategy_descriptor import StrategyDescriptor
+
+
+def string_to_bytes_int(val: str) -> int:
+    if not val:
+        return 0
+    # pylint: disable=consider-using-f-string
+    return int("0x" + "".join("{:02x}".format(ord(c)) for c in val), 16)
```

Comment:
Cannot we use `from starkware.python.utils import from_bytes` instead?

Reply:
Nice, I didn't know about that, thanks.

Reply:
Done in [1a1dc66](https://github.com/software-mansion/protostar/pull/886/commits/1a1dc66b90aed39d3e2737a1131b7e819748495e)